=== modified file 'data/awn.schema-ini.in'
--- data/awn.schema-ini.in	2009-01-31 17:08:34 +0000
+++ data/awn.schema-ini.in	2009-02-01 11:04:32 +0000
@@ -154,7 +154,7 @@
 description = The offset between the arrows and the launchers.
 [app/tasks_have_arrows]
 type = bool
-default = False
+default = True
 description = Determines whether tasks also have arrows beneath them.
 [app/name_change_notify]
 type = bool

