Index: avant-window-navigator/data/awn.schema-ini.in
===================================================================
--- avant-window-navigator.orig/data/awn.schema-ini.in	2009-04-25 01:13:49.000000000 +0200
+++ avant-window-navigator/data/awn.schema-ini.in	2009-04-25 01:13:59.000000000 +0200
@@ -154,7 +154,7 @@
 description = The offset between the arrows and the launchers.
 [app/tasks_have_arrows]
 type = bool
-default = False
+default = True
 description = Determines whether tasks also have arrows beneath them.
 [app/name_change_notify]
 type = bool
