#Ubuntu: 352950
#Upstream : https://bugs.launchpad.net/awn/+bug/352950
=== modified file 'src/awn-task-manager.c'
--- src/awn-task-manager.c	2009-01-31 22:00:56 +0000
+++ src/awn-task-manager.c	2009-04-03 21:37:01 +0000
@@ -1485,7 +1485,7 @@
                         AwnDesktopItem *item, *old_item;
                         item = awn_task_get_item (AWN_TASK (t->data));
                         gchar *file = awn_desktop_item_get_filename (item);
-                        if (strcmp (file, l->data) == 0) {
+                        if (g_strcmp0 (file, l->data) == 0) {
                                 task = AWN_TASK (t->data);
                                 g_free (file);
                                 // refresh the desktop item

